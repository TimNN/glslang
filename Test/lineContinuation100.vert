#version 100

// non-line continuation comment \
#error good error



float f\
oo;  // same as 'float foo;'

#error e2

#define MAIN void main() \
   {                     \
gl_Position = vec4(foo); \
} 

#error e3

MAIN

vec4 foo2(vec4 a)
{                                
  vec4 b = a;       \
  return b;                   
}

// aoeuntheo unatehutna \ antaehnathe 
// anteonuth $ natohe " '
// anteonuth     natohe
/*@*/
/* *@/*/
//@
