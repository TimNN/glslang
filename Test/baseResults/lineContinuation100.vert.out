ERROR: 0:3: 'line continuation' : not supported for this version or the enabled extensions 
ERROR: 0:6: '#error' : e1  
ERROR: 0:8: 'line continuation' : not supported for this version or the enabled extensions 
ERROR: 0:11: '#error' : e2  
ERROR: 0:13: 'line continuation' : not supported for this version or the enabled extensions 
ERROR: 0:14: 'line continuation' : not supported for this version or the enabled extensions 
ERROR: 0:15: 'line continuation' : not supported for this version or the enabled extensions 
ERROR: 0:18: '#error' : e3  
ERROR: 8 compilation errors.  No code generated.

ERROR: node is still EOpNull!
0:20  Function Definition: main( (void)
0:20    Function Parameters: 
0:20    Sequence
0:20      move second child to first child (highp 4-component vector of float)
0:20        'gl_Position' (gl_Position highp 4-component vector of float)
0:20        Construct vec4 (highp 4-component vector of float)
0:20          'foo' (highp float)
0:?   Linker Objects
0:?     'foo' (highp float)

